File: app/Frete/BLL/FreteFaturaBLL.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Emagine.Base.BLL;
using Emagine.Base.Utils;
using Emagine.Frete.Model;
using Frete.Model;

namespace Frete.BLL
{
    public class FreteFaturaBLL: RestAPIBase
    {
        public async Task<List<FreteFaturaInfo>> listar(int idUsuario)
        {
            var url = GlobalUtils.URLAplicacao + "/api/fatura/listar/" + idUsuario.ToString();
            return await queryGet<List<FreteFaturaInfo>>(url);
        }
    }
}

File: www/src/MaisCargas/BLL/FreteFaturaBLL.php
<?php
namespace Emagine\Frete\MaisCargas\BLL;

use Exception;
use Emagine\Frete\MaisCargas\DALFactory\FreteFaturaDALFactory;
use Emagine\Frete\MaisCargas\Model\FreteFaturaInfo;

class FreteFaturaBLL
{
    /**
     * @param $id_usuario
     * @return FreteFaturaInfo[]
     * @throws Exception
     */
    public function listar($id_usuario) {
        $dal = FreteFaturaDALFactory::create();
        return $dal->listar($id_usuario);
    }
}